src/api/blog/content-types/website-post/schema.json

```diff
@@ -73,17 +73,17 @@
       "inversedBy": "websitePosts"
     },
     "cardCover": {
+      "type": "media",
+      "multiple": false,
+      "required": true,
```

Comment:
Since we are updating the cardCover schema... I noticed there are some inconsistencies for allowedType. ousd allows [images, video, files] but story and website only allow [images].

Reply:
Ah good catch - I think we want only `images` here for all 3. I'll add that in.